var/spack/repos/builtin/packages/kadath/package.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack import *
+
+
+class Kadath(CMakePackage):
+    """KADATH SPECTRAL SOLVER.
+
+    The Frankfurt University/Kadath (FUKA) Initial Data solver branch is
+    a collection of ID solvers aimed at delivering consistent initial
+    data (ID) solutions to the eXtended Conformal Thin-Sandwich (XCTS)
+    formulation of Einstein's field equations for a variety of compact
+    object configurations to include extremely compact, asymmetric, and
+    mixed spin binaries.
+    """
+
+    homepage = "https://kadath.obspm.fr/fuka/"
+    git      = "https://gitlab.obspm.fr/grandcle/Kadath.git"
+
+    maintainers = ['eschnett']
+
+    version('fuka', branch='fuka')
+
+    variant('mpi', default=True, description='Enable MPI support')
+
+    variant('codes', multi=True,
+            description="Codes to enable",
+            values=('BBH', 'BH', 'BHNS', 'BNS', 'NS'),
+            default=())
```

Comment:
I believe the unit tests are failing because it's impossible to specify this default from the command line. @alalazo @becker33 what's the recommended approach here, to use `'none'`?